src/main/java/seedu/address/model/team/exceptions/TeamNotFoundException.java

```diff
@@ -3,5 +3,12 @@
 /**
  * Signals that the operation is unable to find the specified team.
  */
+//@@author jordancjq
 public class TeamNotFoundException extends Exception {
+
+    public TeamNotFoundException() {};
+
+    public TeamNotFoundException(String personName) {
```

Comment:
should this be string teamName instead of personName? Since it's checking for teamnotfound